.github/CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 # Guides on how to Contribute
-- [How To Contribute From Your Browser:](Wiki-Contribute-Online) This is currently the easiest way we have to contribute. A simplified version can be found [here](https://medium.freecodecamp.com/how-to-land-your-first-open-source-contribution-from-your-browser-in-15-minutes-756d9bbf81ad) also.
-- [Pull Request Guidelines:](PULL_REQUEST_TEMPLATE) This is a set of points to be checked while making a Pull Request.
-- [How To Contribute To The Wiki:](Wiki-Contribute-Local-GUI) This is a guide that will show you how to have a local copy so you can make changes from your computer, it also works for contributing to the main repository.
-- [More on how to make a pull request from a fork:](Pull-Request-Contribute) This guide has additional information on Pull requests in general and works for contributing to the main repository.
-- [How to Squash Commits:](Git-Squash) We require only one commit per Pull Request unless otherwise specified by a moderator. Please go through this guide to squash multiple commits.
-- How to write a [proper commit message](Git-Commit-Message).
+- [How To Contribute From Your Browser:](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Wiki-Contribute-Online) This is currently the easiest way we have to contribute. A simplified version can be found [here](https://medium.freecodecamp.com/how-to-land-your-first-open-source-contribution-from-your-browser-in-15-minutes-756d9bbf81ad) also.
+- [Pull Request Guidelines:](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/PULL_REQUEST_TEMPLATE) This is a set of points to be checked while making a Pull Request.
+- [How To Contribute To The Wiki:](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Wiki-Contribute-Local-GUI) This is a guide that will show you how to have a local copy so you can make changes from your computer, it also works for contributing to the main repository.
+- [More on how to make a pull request from a fork:](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Pull-Request-Contribute) This guide has additional information on Pull requests in general and works for contributing to the main repository.
+- [How to Squash Commits:](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) We require only one commit per Pull Request unless otherwise specified by a moderator. Please go through this guide to squash multiple commits.
+- How to write a [proper commit message](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Commit-Message).
 
```

Comment:
:+1: for Hard links as it is required when viewing the file in standalone from Wiki repo.